frontend/model/contracts/identity.js

```diff
@@ -20,6 +20,9 @@ DefineContract({
         })
       }),
       process ({ data }, { state }) {
+        Vue.set(state, 'settings', {
+          appLogsFilter: ['error', 'warn']
+        })
```

Comment:
Let's keep these settings unique to each client device by storing them in the vuex root settings instead.